### PR TITLE
Care Card component 

### PR DESCRIPTION
**Pull request by chrimesdev**
## Description
Refactoring the Care Card component, creating the macro for use on example pages and adding some BackstopJS testing. Updated all the example pages to use the Care Card component macro and deleted some older example pages we no longer really use. 

Published the Care Card component into the **/dist/packages/** folder along with the Action Link component. 

## Checklist
- [ ] Design and code examples added to the Design System pages
- [ ] Changes added to [CHANGELOG.md](../CHANGELOG.md)
- [ ] Successful linting of the SCSS (`npm run lint-css`)

### CODE REVIEW

File: app/views/macros/care-card.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<div class="nhsuk-c-care-card nhsuk-c-care-card--grey">
+{% elseif type == 'red' %}
+<div class="nhsuk-c-care-card nhsuk-c-care-card--red">
+{% elseif type == 'black' %}
+<div class="nhsuk-c-care-card nhsuk-c-care-card--black">
+{% else %}
+<div class="nhsuk-c-care-card">
+{% endif %}
+  <div class="nhsuk-c-care-card__heading">
+    <h4 class="nhsuk-heading-m">{{title}}</h4>
```
Comment (mcheung-nhs) [inline app/views/macros/care-card.html]: Are call card headings going to be H4 or H3?

File: app/views/styles/care-cards.html [hunk 2]
```diff
@@ -9,12 +10,15 @@ <h1>Care Cards</h1>
...
 
 {% block sidebar %}
-    <h2 class="nhsuk-heading-xs">Contents</h2>
-    <ul class="">
-		<li><a href="#care_card_111">111 Care Card</a></li>
-		<li><a href="#care_card_primary">Primary Care Card</a></li>
-		<li><a href="#care_card_urgent">Urgent Care Card</a></li>
-		<li><a href="#care_card_emergency">Emergency Care Card</a></li>
+    <h2 class="nhsuk-heading-xs">Page contents</h2>
+    <ul>
```
Comment (mcheung-nhs) [inline app/views/styles/care-cards.html:14]: Don't know what to do about these internal links as they don't go anywhere? Not a showstopper.

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

File: app/views/macros/care-card.html (hunk 1 above)
Comment (chrimesdev) [reply app/views/macros/care-card.html]: Fixed

File: app/views/styles/care-cards.html (hunk 2 above)
Comment (chrimesdev) [reply app/views/styles/care-cards.html:14]: I've just added those as placeholders I'm hoping David or Ben will update the content of these pages following this.

**Review by mcheung-nhs — APPROVED** [review]

**Pull request merged**